docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -18,59 +18,79 @@ _([npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7
 The output should look like this:
 
 ```bash filename="Terminal"
-Usage
-  $ next <command>
-
-Available commands
-  build, start, export, dev, lint, telemetry, info
-
-Options
-  --version, -v   Version number
-  --help, -h      Displays this message
-
-For more information run a command with the --help flag
-  $ next build --help
+Usage next [options] [command]
+
+The Next.js CLI allows you to develop, build, start your application, and more.
+
+Options:
+  -v, --version                Outputs the Next.js version.
+  -h, --help                   Displays this message.
+
+Commands:
+  build [directory] [options]  Creates an optimized production build of your application.
+                               The output displays information about each route.
+  dev [directory] [options]    Starts Next.js in development mode with hot-code reloading,
+                               error reporting, and more.
+  info [options]               Prints relevant details about the current system which can be
+                               used to report Next.js bugs.
+  lint [directory] [options]   Runs ESLint for all files in the `/src`, `/app`, `/pages`,
+                               `/components`, and `/lib` directories. It also provides a
+                               guided setup to install any required dependencies if ESLint
+                               is not already configured in your application.
+  start [directory] [options]  Starts Next.js in production mode. The application should be
+                               compiled with `next build` first.
+  telemetry [options]          Allows you to enable or disable Next.js' completely
+                               anonymous telemetry collection.
 ```
 
 You can pass any [node arguments](https://nodejs.org/api/cli.html#cli_node_options_options) to `next` commands:
 
 ```bash filename="Terminal"
-NODE_OPTIONS='--throw-deprecation' next
-NODE_OPTIONS='-r esm' next
-NODE_OPTIONS='--inspect' next
+NODE_OPTIONS='--throw-deprecation' npx next
+NODE_OPTIONS='-r esm' npx next
+NODE_OPTIONS='--inspect' npx next
```

Comment:
```suggestion
NODE_OPTIONS='--throw-deprecation' next
NODE_OPTIONS='-r esm' next
NODE_OPTIONS='--inspect' next
```
I don't think we want to show `npx` here as that might cause users to have a different version run from their locally installed version